src/io/flutter/run/SdkRunner.java

```diff
@@ -22,6 +28,18 @@ public String getRunnerId() {
     return "FlutterRunner";
   }
 
+  @Override
+  protected RunContentDescriptor doExecute(@NotNull RunProfileState state, @NotNull ExecutionEnvironment env) throws ExecutionException {
+    if (FlutterUtils.is2018_3_or_higher()) {
+      // Force "allow running in parallel" (see: #2875).
+      final RunnerAndConfigurationSettings settings = env.getRunnerAndConfigurationSettings();
+      if (settings != null) {
+        settings.setSingleton(false);
```

Comment:
Very cool! Does this address the issue for you - we're you then able to hot restart by clicking on the run button?

I'll patch this in locally.